client/lib/abtest/active-tests.js

```diff
@@ -128,7 +128,7 @@ export default {
 		localeTargets: 'any',
 	},
 	sidebarUpsellNudgeUnification: {
-		datestamp: '20200127',
+		datestamp: '20200221',
```

Comment:
@martinremy is it okay if we reuse the test name here?

Reply:
Yep, you can reuse the test name with a new date, that will give us a unique slug for new Hypotheses.